src/data-tutorials/users.tsx

```diff
@@ -805,6 +805,13 @@ const Users: User[] = [
     website: '/blog/tutorials/cryptography-in-blockchain-an-overview-of-hash-functions-and-digital-signatures',
     tags: ['celosage', 'intermediate'],
   },
+  {
+    title: 'How to build a Basic CRUD App in Solidity',
+    description: 'This post will teach you how to create a basic Create, Read, Update and Delete (CRUD) smart contract and test it with the Truffle framework.',
+    preview: require('./showcase/celo-sage.png'),
```

Comment:
Change preview to

`preview: require('./showcase/beginner/how-to-build-a-basic-crud-app-in-solidity.png')`